### DISCUSSION TITLE
Edit mp3 tag

### DISCUSSION THREAD

**Discussion by drazde** [cmus/cmus#1141] [General]
Is it possible edit mp3 tag with cmus?
If no I think it could be a great features ;-)

**Comment by nefthy** [comment]
It goes against the philosophy of cmus, to modify audio files in any way. You will have to use an external tool for editing id3.

It is possible to bind a key that will open an id3 editor of your choice for the current file. Something along the lines:

`:bind 0 common run easytag {}`

**Reply by hikikomori4** [reply]
Error: invalid context '0'

**Comment by drazde** [comment]
Ok thanks
